RELEASING.rst

```diff
@@ -0,0 +1,55 @@
+Releasing sphinxcontrib-images
+==============================
+
+1. Test releasing to TestPyPI
+
+   - Bump the package version by editing ``sphinxcontrib/images.py``
+     (i.e. from 0.9.2 to 0.9.3.rc1 -- use .rcN for subsequent releases),
```

Comment:
Since 0.9.3.rc1 is normalized to 0.9.3rc1 (https://peps.python.org/pep-0440/#pre-release-separators) should we just use 0.9.3rc1?